aws-rds-dbinstance/src/main/java/software/amazon/rds/dbinstance/Translator.java

```diff
@@ -482,17 +483,17 @@ public static String translateDBParameterGroupFromSdk(final software.amazon.awss
 
     public static Map<String, String> translateTagsToRequest(final Collection<Tag> tags) {
         return streamOfOrEmpty(tags)
-                .collect(Collectors.toMap(Tag::getKey, Tag::getValue, (v1, v2) -> v2));
+                .collect(Collectors.toMap(Tag::getKey, Tag::getValue, (v1, v2) -> v2, LinkedHashMap::new));
     }
 
-    public static List<software.amazon.awssdk.services.rds.model.Tag> translateTagsToSdk(final Collection<Tag> tags) {
+    public static Set<software.amazon.awssdk.services.rds.model.Tag> translateTagsToSdk(final Collection<Tag> tags) {
         return streamOfOrEmpty(tags)
                 .map(tag -> software.amazon.awssdk.services.rds.model.Tag.builder()
                         .key(tag.getKey())
                         .value(tag.getValue())
                         .build()
                 )
```

Comment:
Should we sort here to ensure same order between all handler responses?
```
.sorted((t1, t2)->o1.getKey().compareTo(o2.getKey()))
```

Reply:
I believe we would rather eliminate the order change, i.e. keep it "as given"